### ISSUE TITLE
No module named requests error when trying to share unlisted mods

### ISSUE THREAD

**Issue by E-Gamma-102** [zzzz465/Rimworld-automatic-mod-sorter#12]
Critical Error occured while running...
 No module named 'requests'
please report this to issue in github

**Comment by Rescura** [comment]
Yes, having the same issue when I try to upload the DB log.. 'requests' python module isn't installed in dll I guess?
